.github/workflows/package.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: Set up MinGW
         if: runner.os == 'Windows'
-        uses: egor-tensin/setup-mingw@84c781b557efd538dec66bde06988d81cd3138cf
+        uses: egor-tensin/setup-mingw@v2
```

Comment:
Let's always use commit hashes for non-officla actions
We can also try and test this change by running workflow to test pypi - please do so